src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -250,6 +252,22 @@ public void testPullPrivateRepoWithoutAuth() throws Exception {
   }
 
   @Test
+  public void testBuildImageIdWithBuildargs() throws Exception {
+    assumeTrue("Docker API should be at least v1.21 to support buildargs while building, got "
```

Comment:
Sorry ignore my two comments above. I see what you're doing now. I read through too hastily the first time.